templates/procuration/landing.html.twig

```diff
@@ -22,37 +22,39 @@
-                        <div class="b__nudge--bottom text--summary">
-                            Je veux voter au nom d'un(e) citoyen(ne) de ma ville.
-                        </div>
-                        <a href="{{ path('app_procuration_choose_election', { action: constant('AppBundle\\Procuration\\ElectionContext::ACTION_PROPOSAL') }) }}" class="btn btn--primary">
-                            Je me porte mandataire
-                        </a>
-                    </div>
-                    <div>
-                        <h2 class="text--medium text--bold b__nudge--bottom-small">
-                            Je ne suis pas présent(e) le 26 mai.
-                        </h2>
-                        <div class="b__nudge--bottom text--summary">
-                            Un(e) habitant(e) de ma ville va voter en mon nom.
+                <section class="procuration__content l__wrapper--500 l__col space--60-0 text--center">
+                    {% if next_election %}
+                        <div class="b__nudge--bottom-60">
+                            <h2 class="text--medium text--bold b__nudge--bottom-small">
+                                Je suis présent(e) le 26 mai.
+                            </h2>
+                            <div class="b__nudge--bottom text--summary">
+                                Je veux voter au nom d'un(e) citoyen(ne) de ma ville.
+                            </div>
+                            <a href="{{ path('app_procuration_choose_election', { action: constant('AppBundle\\Procuration\\ElectionContext::ACTION_PROPOSAL') }) }}" class="btn btn--primary">
+                                Je me porte mandataire
+                            </a>
                         </div>
+                        <div>
+                            <h2 class="text--medium text--bold b__nudge--bottom-small">
+                                Je ne suis pas présent(e) le 26 mai.
+                            </h2>
+                            <div class="b__nudge--bottom text--summary">
+                                Un(e) habitant(e) de ma ville va voter en mon nom.
+                            </div>
 
-                        <a href="{{ path('app_procuration_choose_election', { action: constant('AppBundle\\Procuration\\ElectionContext::ACTION_REQUEST') }) }}" class="btn btn--blue">
-                            Je donne procuration
-                        </a>
-                    </div>
-                {% else %}
-                    No coming election, TODO.
-                {% endif %}
-            </section>
+                            <a href="{{ path('app_procuration_choose_election', { action: constant('AppBundle\\Procuration\\ElectionContext::ACTION_REQUEST') }) }}" class="btn btn--blue">
+                                Je donne procuration
+                            </a>
+                        </div>
+                    {% else %}
+                        No coming election, TODO.
```

Comment:
`TODO.` can not be updated or removed ? :thinking:

Reply:
Good question ^^
by the way, this code is not from me